### PR TITLE
Publish with `next`tag

### PR DESCRIPTION
**Pull request by gziolo**
Executed `npm run publish:dev` to publish new packages with `next` tag:

- @wordpress/a11y@1.0.4
 - @wordpress/autop@1.0.2
 - @wordpress/babel-preset-default@1.0.1-0
 - @wordpress/dom-ready@1.0.1
 - @wordpress/hooks@1.1.2
 - @wordpress/jest-console@1.0.1
 - @wordpress/url@1.0.1

### CODE REVIEW

File: packages/babel-preset-default/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/babel-preset-default",
-	"version": "1.0.0",
+	"version": "1.0.1-0",
```
Comment (ntwb) [inline packages/babel-preset-default/package.json]: @gziolo `1.0.1-0`? Was this meant to be `1.0.1`?

Comment (gziolo) [reply packages/babel-preset-default/package.json]: I wanted to check what happens when i reference this preset from the `package.json` in the root folder. Unfortunately when I do it, it doesn't use the latest code, rather published version from the npm. I will continue using relative path to `./packages/babel-preset-default`. I will publish stable version as `1.0.0`. This was pre-release tag.

Comment (ntwb) [reply packages/babel-preset-default/package.json]: Right 👍, `.babelrc` isn't taking precedence over `package.json` is it?

Comment (gziolo) [reply packages/babel-preset-default/package.json]: I don’t remember, they explain it in their docs 😃

**Pull request merged**